Core/clim-core/presentation-defs.lisp

```diff
@@ -1382,8 +1387,11 @@ protocol retrieving gestures from a provided string."))
 
 (define-presentation-method accept ((type real) stream (view textual-view)
                                     &key)
-  (let ((*read-base* base))
-    (accept-using-read stream type)))
+  (let ((*read-base* base)
+	(read-result (accept-using-read stream type)))
+    (if (and (null read-result) default)
+	(values default type)
```

Comment:
Oh ffs. I'm going to re-create the repo, apply the changes again so there don't have to be extra commits in master.

What 'indenter' do you use? The only reason we're here is because I'm typing s-expressions out by hand rather than copy-pasting my ALREADY WORKING ones into climacs, which indents them differently than you like.